src/gepetto_viewer_rerun/entity.py

```diff
@@ -0,0 +1,14 @@
+import rerun as rr
+from dataclasses import dataclass
+
+
+@dataclass
+class Entity:
+    def __init__(self, name: str, archetype: rr.archetypes):
+        self.name = name
+        self.archetype = archetype
```

Comment:
```suggestion
    name: str
    archetype: rr.archetypes
```

This is how dataclasses work